gatsby-node.js

```diff
@@ -349,41 +349,17 @@ const BuildPage = (page, actions) => {
 exports.onCreatePage = ({ page, actions }) => {
     const { deletePage } = actions
     const isProduction = process.env.GATSBY_ENV === 'production'
-    const pagesToBuild = process.env.GATSBY_BUILD_PAGES || 'all'
+    const pagesToBuild = process.env.GATSBY_BUILD_PAGES
```

Comment:
is this an array or string?

Reply:
string I guess, but it will be great if this can handle multiple pages like `markets | help-centre`

Reply:
Yeah, that's what i am thinking. if it's string then we can't build more than one page.

Reply:
Its a string.I don't think so the developer will work on multiple pages at a time.They can access the page by changing the route.But yeah we can add for multiple pages.But it should not increase the memory heap

Reply:
Good callout @hasan-deriv 
It has to allow multiple pages.

Reply:
I agree to have multiple pages , sometimes one component is shared and can affect on other pages also

Reply:
Agreed @mitra-deriv